src/ir/ir.jl

```diff
@@ -625,7 +625,9 @@ new variable. See also [`pushfirst!`](@ref), [`insert!`](@ref).
       %2 = %1 * %1
 """
 function push!(b::Block, x::Statement)
-  x = applyex(a -> push!(b, Statement(x, expr = a)), x)
+  if !isexpr(x.expr, :foreigncall) # needed to avoid #30
```

Comment:
```suggestion
  if !isexpr(x.expr, :foreigncall) # needed to avoid https://github.com/MikeInnes/IRTools.jl/issues/30
```
I think a link is usually more convenient/obvious (especially on the small chance the repo moves)